test/v5/kyberTradeLogic.js

```diff
@@ -19,8 +19,10 @@ const negligibleRateDiffBps = new BN(5); //0.05%
 const maxDestAmt = new BN(2).pow(new BN(255));
 const minConversionRate = new BN(0);
 
-let takerFeeBps = new BN(20);
-let platformFeeBps = new BN(0);
+let takerFeeArray = [new BN(0), new BN(250), new BN(400)];
+let platformFeeArray = [new BN(0), new BN(250, new BN(400))];
```

Comment:
I think its more readable if these 2 arrays above are near the loop.
and not here.

lets think of tests as separate from one another.
its clearer for the reader and runner of tests.

so if I decide to run only a specific test and if fails.
all relevant parameters are near by.